Pipfile

```diff
@@ -5,22 +5,22 @@ name = "pypi"
 
 [packages]
 bcrypt = "4.0.1"
-cryptography = "==42.0.8"
-pyOpenSSL = "==24.1.0"
+cryptography = "==43.0.1"
 distro = "==1.8.0"
 jsonschema = "==4.20.0"
 pexpect = "==4.9.0"
 ply = "==3.11"
 pyasn1 = "==0.5.0"
 toml = "==0.10.2"
 yappi = "==1.6.0"
-setuptools = "69.5.1"
-aiohttp = "==3.9.5"
+setuptools = "70.0.0"
+aiohttp = "==3.10.5"
 python-dateutil = "2.8.2"
 msgpack = "1.0.7"
 parameterized = "*"
 typing-extensions = "*"
 asyncssh = "*"
+pyopenssl = "==24.2.1"
```

Comment:
move pyopenssl back next to cryptography.

pyopenssl depends heavily on cryptography. Let's keep all the encryption-related dependencies next to each other, i.e., bcrypt.